en/components/date_time_editor.md

```diff
@@ -0,0 +1,175 @@
+```html
+<igx-input-group>
+    <input type="text" igxInput igxDateTimeEditor [(ngModel)]="date"/>
+</igx-input-group>
+```
+<div class="divider--half"></div>
+
+
+#### Keyboard Navigation
+Date Time Editor Directive has intuitive keyboard navigation that makes it easy to jump through different DateParts, increment, decrement, etc. without having to touch the mouse.
+
+- <kbd>Ctrl</kbd> / <kbd>Cmd</kbd> + <kbd>Arrow Left</kbd> / <kbd>Right</kbd> - navigates between date sections. Jumps from current section to the same caret position of the next/previous section. On start/end sections, first jump to the other end of the same section following the direction of the arrow.
+
+- <kbd>Arrow Up</kbd> / <kbd>Down</kbd> - increment/decrement date portions. See related [`isSpinLoop`]({environment:angularApiUrl}/classes/igxdatetimeeditordirective.html#isspinloop)
+
+- <kbd>Ctrl</kbd> / <kbd>Cmd</kbd> + <kbd>;</kbd> - set the current day and time in the editor via a shortcut.
+
+
+### Examples
+
+#### Display and input format
+IgxDateTimeEditor support different display and input formats.
+
+For display, it uses Angular's [DatePipe](https://angular.io/api/common/DatePipe) which allows it to support pre-defined format options, such as `shortDate` and `longDate`. It can also accept a constructed format string using characters supported by DatePipe, e.g. `EE/MM/yyyy`. Notice formats like shortDate, longDate, etc., can be used as `displayFormat` only.
+
+To set specific input format, pass it as string to the IgxDateTimeEditor directive. This will set both the expected user input format and [mask](mask.md) for the editor. 
+
+
+```html
+<igx-input-group>
+    <input type="text" igxInput [displayFormat]="'shortDate'" [igxDateTimeEditor]="'dd/MM/yyyy'" [(ngModel)]="date"/>
+</igx-input-group>
+```
+
+#### Min max value
+You can specify [`minValue`]({environment:angularApiUrl}/classes/igxdatetimeeditordirective.html#minvalue) and [`maxValue`]({environment:angularApiUrl}/classes/igxdatetimeeditordirective.html#maxvalue) properties to restrict input and control the validity of the ngModel.
+```typescript
+public minDate = new Date(2020, 1, 15);
+public maxDate = new Date(2020, 12, 1);
+``` 
+
+```html
+<igx-input-group>
+    <input type="text" igxInput igxDateTimeEditor [minValue]="minDate" [maxValue]="maxDate" [(ngModel)]="date" required/>
+</igx-input-group>
+
+```
+
+#### Increment and decrement
+`igxDateTimeEditor` directive exposes public `increment` and `decrement` methods. These increment/decrement a specific `DatePart` of the currently set date and time and can be used in a couple of ways.
+
+In the first scenario, if no specific DatePart is passed to the method, a default DatePart will increment/decrement, based on the specified `inputFormat` and the internal directive implementation.
+In the second scenario, you can explicitly specify what DatePart to manipulate as it may suite different requirements.
+You may compare both in the following sample:
```

Comment:
we may add some code here showing how to increment and decrement the dates. Otherwise any one should open the Stackblitz and should find where this is happening

Reply:
As far as I remember, it was decided to not duplicate code/snippets/sample. In this case the  comparison is available in the sample for both functionality/visually and code.
@damyanpetev what do you think. How do you feel adding  typescript snippets for increment/decrement ?